src/main/java/hudson/scm/IntegritySCM.java

```diff
@@ -579,15 +582,11 @@ private void writeChangeLog(Run<?, ?> run, TaskListener listener, File changeLog
    * @param run
    * @param listener
    * @param siProject
-   * @throws APIException
-   * @throws AbortException
-   * @throws IOException
-   * @throws InterruptedException
    * @throws InterruptedException
+   * @throws Exception 
    */
   private void checkPointBeforeBuild(Run<?, ?> run, TaskListener listener,
-      IntegrityCMProject siProject) throws AbortException, IOException, InterruptedException,
-          com.mks.api.response.InterruptedException, APIException
+      IntegrityCMProject siProject) throws Exception

```

Comment:
It is a bad design practice. You have replaced a method with the documented exception list to a wide-open report